app/util/links.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+export const external = {
+  cloudInitFormat: 'https://cloudinit.readthedocs.io/en/latest/explanation/format.html',
+  cloudInitExamples:
+    'https://cloudinit.readthedocs.io/en/latest/reference/examples.html#yaml-examples',
+}
+
+const links = {
+  external,
+}
+
+export default links
```

Comment:
I should put in a lint rule or something about this — I think the best practice is to avoid default exports. Bundlers don't like them, something like that. They don't really do anything useful anyway. For now you can also collapse this down, we don't need to distinguish external. I'd just do `export const links = { cloudInitFormat: ... }`.